src/features/hotspots/details/TimelinePicker.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '../../../components/Box'
+import Bars from '../../../assets/images/bars.svg'
+import Text from '../../../components/Text'
+import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import CarotDown from '../../../assets/images/carot-down.svg'
+import ModalPicker from '../../../components/ModalPicker'
+
+type Props = {
+  onTimelineChanged: (value: string, index: number) => void
+}
+
+const TimelinePicker = ({ onTimelineChanged }: Props) => {
+  const { t } = useTranslation()
```

Comment:
If you wanted, you could make this dynamic by creating an array in en.ts
```
    picker_options: ['24 Hours', '7 Days', '14 Days', '30 Days'],
```

and then use it like this
```
const options: Array<string> = t('hotspot_details.picker_options', {
    returnObjects: true,
})
```

Reply:
Nice!